docs/intro/tutorial08.txt

```diff
@@ -0,0 +1,131 @@
+package. From 2020-2022, the Django Debug Toolbar has ranked in the top three
+most used third-party packages according to the Django Developer Survey (
+`2022 <https://lp.jetbrains.com/to-be-released-yet>`_,
+`2021 <https://lp.jetbrains.com/django-developer-survey-2021-486/#horizontal-bar-chart-109>`_,
+`2020 <https://docs.google.com/forms/d/15GRScagj29AjfFh1Js0ujCEQ2EUqMer5nMB3Fp1KoG8/viewanalytics#21>`_,
+).
+
+.. admonition:: Where to get help:
+
+    If you're having trouble going through this tutorial, please head over to
+    the :doc:`Getting Help</faq/help>` section of the FAQ.
+
+Installing the Django Debug Toolbar
+===================================
+
+A useful tool for debugging Django web applications is the Django Debug
+Toolbar. It's a third-party package maintained by the `Jazzband
+<https://jazzband.co>`_ organization. The toolbar helps you understand how your
+application functions and to identify problems. It does so by providing panels
+that provide debug information about the current request/response.
+
+- Cache panel: Presents you with the cache operations made and the context
+  of the executions.
+- Profiler panel: Presents you with a `cProfiler
+  <https://docs.python.org/3/library/profile.html>`_ summary of the codes
+  execution.
+- Request panel: Presents you with the headers and body of a request. This
+  allows you to determine what values are being sent to the Django application.
+- SQL panel: Presents you with all
+  the SQL queries made during the lifespan of a request.
+- Templates panel: Presents you with the templates used when rendering the
+  response and the context used to render them.
+
+To install a third-party application like the toolbar, you need to install
+the package by running the command within an activated virtual environment.
+This is like our earlier step of :ref:`installing an official release of
+Django <installing-official-release>`.
+
+   .. console::
+
+        $ python -m pip install django-debug-toolbar
+
+.. admonition:: Managing application dependencies
+
+    Managing dependencies is an important part developing an application, but
+    it is outside of the scope of this tutorial. Please see the official Python
+    documentation regarding `Managing Application Dependencies
+    <https://packaging.python.org/en/latest/tutorials/managing-dependencies/>`_
+    for more information.
+
+Third-party packages that integrate with Django may need some setup.
+Sometimes you will need to add the package's Django app to your
+:setting:`INSTALLED_APPS` setting. `The Django Debug Toolbar has many
+installation steps
+<https://django-debug-toolbar.readthedocs.io/en/latest/installation.html>`_.
+We're linking to the toolbar's documentation here because as a third-party
+package, it's maintained outside of the core development of Django. This
+allows the package maintainers to make releases outside of Django's release
+schedule
```

Comment:
Split into two paragraphs - one introducing the concept of setup, the second linking to DDT instructions

```suggestion
Third-party packages that integrate with Django need some post-installation
setup to integrate them with your project. Often you will need to add the
package's Django app to your :setting:`INSTALLED_APPS` setting. Some packages
need other changes, like additions to your URLconf (``urls.py``).

Django Debug Toolbar requires several setup steps. Follow them in `its
installation guide
<https://django-debug-toolbar.readthedocs.io/en/latest/installation.html>`_.
The steps are not duplicated in this tutorial, because as a third-party
package, it may change separately to Django's schedule.
```